packages/inspector/style/base.css

```diff
@@ -38,11 +38,7 @@
   white-space: pre-wrap;
 }
 
-.jp-Inspector-default-content {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-size: xx-large;
-  font-style: italic;
-  color: darkgray;
+.jp-Inspector-placeholderContent {
```

Comment:
Can you mention the class name change in the [Extension Migration Guide](https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#css-styling)?

Reply:
Added! What do you think?

<img width="1582" alt="Screenshot 2024-07-11 at 08 10 35" src="https://github.com/jupyterlab/jupyterlab/assets/17132927/1cf007b2-2a14-48d1-91c7-b3dd0585ef59">